pkg/cli/registry/login/login.go

```diff
@@ -126,6 +132,17 @@ func NewRegistryLoginCmd(name string, f kcmdutil.Factory, streams genericcliopti
 }
 
 func (o *LoginOptions) Complete(f kcmdutil.Factory, args []string) error {
+	credentials := 0
+	if len(o.ServiceAccount) > 0 {
+		credentials++
+	}
+	if len(o.AuthBasic) > 0 {
+		credentials++
+	}
+	if credentials > 1 {
+		return fmt.Errorf("You may only specify a single authentication input as -z or --auth-basic")
```

Comment:
lower case error

Reply:
This is an end user message from the top of the command so it is capitalized like other usage errors